docs/dev/zksync-v2/handling-of-eth.md

```diff
@@ -1,7 +1,7 @@
 # Handling of ETH and tokens
 
-zkSync has no "native" token, and the fees can be paid in ETH or ERC20 tokens, but in order to have higher compatibility with Ethereum, we decided to preserve some special behaviour of ether.
+zkSync 1.x allows users to pay fees with ERC20 tokens. The previous versions of the testnet allowed that too, but with the advent of the [paymaster](./aa.md#paymasters) feature it has become redundant. To provide better compatibility and alignment with the L1, zkSync 2.0 allows only Ether as a fee token.
 
-On zkSync, `ETH` is a special token with address `0x000000000000000000000000000000000000800a`, though zero address (`0x0000000000000000000000000000000000000000`) is often used as an alias for it. It can only be transferred by providing `msg.value`, i.e. the same way it is done on Ethereum and it does _not_ expose the ERC20 contract interface.
+`ETH` is implemented a special token with address `0x000000000000000000000000000000000000800a`, though zero address (`0x0000000000000000000000000000000000000000`) is often used as an alias for it. It can only be transferred by providing `msg.value`, i.e. the same way it is done on Ethereum and it does _not_ expose the ERC20 contract interface.
```

Comment:
```suggestion
`ETH` is implemented as a special token with address `0x000000000000000000000000000000000000800a`, though zero address (`0x0000000000000000000000000000000000000000`) is often used as an alias for it. It can only be transferred by providing `msg.value`, i.e. the same way it is done on Ethereum and it does _not_ expose the ERC20 contract interface.